### ISSUE TITLE
All images, CAPTCHA elements, and other non-text content has an accurate text-based alternative.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#138]
Several images in the "Purdue News on YouTube" section, such as "Purdue Experts" have inaccurate aria-label (e.g., "https://marcom.purdue.edu/app/uploads/2023/08/Purdue-Experts.jpg"). This description doesn't convey the image's purpose. Either provide meaningful accessible text for each image or hide these images completely from screen-reader users.

The decorative chevrons of "Read More" links should be hidden from screen-reader users. VoiceOver users currently encounter these as images without accessible names.

**Comment by justpie** [comment]
The images are decorative and don't need labels.
